.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -506,7 +506,9 @@ instance has label `azure/foo=bar`.
 
 ### Passwordless
 
-This feature has additional build requirements, so it should be tested with a pre-release build from Drone (eg: `https://get.gravitational.com/teleport-v10.0.0-alpha.2-linux-amd64-bin.tar.gz`).
+This feature has additional build requirements, so it should be tested with a
+pre-release build from Drone (eg:
+`https://get.gravitational.com/tsh-v10.0.0-alpha.2.pkg`).
```

Comment:
I think this makes more sense, as the macOS tsh is actually the "difficult" binary, others don't need anything special to work.

Reply:
good idea. maybe also update the version to `v14` while we are at it?

Reply:
I don't think it's worth it. By the time anyone reads this copy, the version will have changed again.